src/__tests__/__snapshots__/Flex.js.snap

```diff
@@ -23,9 +23,9 @@ exports[`Flex justify prop 1`] = `
   display: -webkit-flex;
   display: -ms-flexbox;
   display: flex;
-  -webkit-box-pack: justify;
+  -webkit-box-pack: space-between;
   -webkit-justify-content: space-between;
-  -ms-flex-pack: justify;
+  -ms-flex-pack: space-between;
```

Comment:
I believe this is the same snapshot/dependency issue that @chrisdroukas was running into. Make sure you `rm -rf node_modules && npm i && npm t -- -u` here